Amqp.V1/Akka.Streams.Amqp.V1/Dsl/AmpqSink.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Akka.Streams.Dsl;
+using System.Threading.Tasks;
+
+namespace Akka.Streams.Amqp.V1.Dsl
+{
+    public static class AmpqSink<T>
+    {
+        public static Sink<T, Task> Create(IAmpqSinkSettings<T> sourceSettings)
```

Comment:
I would move the `T` into the method definition, in theory you should be able to call `AmpqSink.Create(settings)` because the `T` could be inferred from the settings.